content/documentation/programming-guide/index.html

```diff
@@ -2429,8 +2429,7 @@ <h3 id="default-coders-and-the-coderregistry">6.2. Default coders and the CoderR
 
 <h4 id="default-coder-lookup">6.2.1. Looking up a default coder</h4>
 
-<p class="language-java">You can use the method <code class="highlighter-rouge">CoderRegistry.getDefaultCoder</code> to determine the default
-Coder for a Java type. You can access the <code class="highlighter-rouge">CoderRegistry</code> for a given pipeline
+<p class="language-java">You can access the <code class="highlighter-rouge">CoderRegistry</code> for a given pipeline
```

Comment:
I think the method name just changed to `CoderRegistry.getCoder`:
https://github.com/apache/beam/blob/73cc33292dee500c0e9f1a072f4d516f753f8e9d/sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java#L244

Please just swap `getDefaultCoder` with `getCoder`

Reply:
Fixed. Thanks @lukecwik